src/Billing/Entities/PaymentCard.php

```diff
@@ -13,20 +13,32 @@
  * @property string $postcode
  * @property string $cardNumber
  * @property string $cardType
- * @property DateTime $validFrom
- * @property DateTime $expiry
+ * @property string $validFrom
+ * @property string $expiry
  * @property int $issueNumber
  * @property boolean $primaryCard
  */
 class PaymentCard extends Entity
 {
-    protected $dates = ['validFrom', 'expiry'];
-
+    /**
+     * Check if the PaymentCard has expired.
+     *
+     * @return bool
+     * @throws \Exception
+     */
     public function isExpired()
     {
-        return $this->expiry < new DateTime();
+        $date = DateTime::createFromFormat('m/y', $this->expiry);
+
+        return $date->format('Y-m-t') < (new DateTime())->format('Y-m-d');
```

Comment:
Why are you comparing them as strings?

Surely you want to compare them as DateTime objects?